lib/emscripten/src/lib.rs

```diff
@@ -213,17 +221,33 @@ impl<'a> EmscriptenData<'a> {
             dyn_call_viji,
             dyn_call_vijiii,
             dyn_call_vijj,
+            #[cfg(feature = "vfs")]
+            vfs: None,
         }
     }
 }
 
 pub fn run_emscripten_instance(
-    _module: &Module,
+    module: &Module,
     instance: &mut Instance,
     path: &str,
     args: Vec<&str>,
 ) -> CallResult<()> {
     let mut data = EmscriptenData::new(instance);
+
+    // Construct a new virtual filesystem and inject it into the emscripten data
+    // This is behind a feature flag for now, but will be default in the future
+    #[cfg(feature = "vfs")]
+    {
+        data.vfs = match module.info().custom_sections.get("wasmer_fs") {
```

Comment:
Can we use `wasmer:fs` as the custom section name, so that it's clear that `wasmer` is a namespace?

Reply:
I like `wasmer:fs` also more than `wasmer_fs` (as it namespace it more clearly)

Reply:
Ok I will make this correction...